### ISSUE TITLE
Build out the intubations infographics screen

### ISSUE THREAD

**Issue by lukesleeman** [Western-Health-Covid19-Collaboration/wh_covid19_app#39]
From the main menu there is a button which takes you to the intubation infographics screen.  That screen is currently **really** basic.

**We need to build build out a detailed version of the intubation infographics screen**

@greggmiller Can you please provide some information in this issue on what should be in this section?  Once that is done assign the issue to Marc Edwards.

@marcedwards Once greg has provided info, can you provide some design guidance in this issue in on how it should look.  Once this is done, can you label this issue as "Ready for Dev" so that one of the devs can pick it up.

**Comment by greggmiller** [comment]
Please refer document SAS COVID 19 infographics 1 2.pdf

This is pages 1 and 2, might need a slide option to go between the two and a zoom.

Thanks!

**Comment by lukesleeman** [comment]
[SAS COVID 19 infographics 1 2.pdf](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/files/4386285/SAS.COVID.19.infographics.1.2.pdf)

**Comment by lukesleeman** [comment]
I spent some time discussing this issue and issue #37 with @Mattro on facetime tonight.

It seems like this screen and the "intubation step by step guidance screen" are duplicates of each other?  It's basically that we need to "SAS COVID 19 infographics 1 2.pdf" into the app?

I've asked @Mattro to progress the work via issue #37 and I'm reassigning this one over to @greggmiller 

Greg:  Can you clarify how the "intubations algorithm screen" is different to the "intubation step by step guidance screen"?  If we just have our wires crossed and they are exactly the same thing, can you close this issue?

**Comment by greggmiller** [comment]
Page 2 of SAS COVID 19 infographics 1 2 is not covered in step-by-step guidance so is worth including here.

Given that this topic is now "Airway" could we change this section form "Algorithm" to "Infographics" as we now have infographics that we are using for "Intubation" and "Extubation" from different society/guidelines.

The intention would be to allow us to have quick access to all of these infographics. 


Let me know @galtaforce if there are any other inforgraphics on airway  I may have missed.

[Extubation_Infographic.pdf](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/files/4386891/Extubation_Infographic.pdf)
[SAS COVID 19 infographics 1 2.pdf](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/files/4386892/SAS.COVID.19.infographics.1.2.pdf)

Given all this and multiple other changes through the day I will log a new issue describing the proposed changes in hierarchy and home screen and tag @Western-Health-Covid19-Collaboration/designers .

**Comment by lukesleeman** [comment]
Ok, thats great.  So just to be 100% clear:

Issue #37 **only covers page 1 of `SAS COVID 19 infographics 1 2.pdf`**

This issue is that we should display page two of `SAS COVID 19 infographics 1 2.pdf` and `Extubation_Infographic.pdf` just as pinch to zoom images, just so we have the infographics in the app?

@greggmiller Is that correct?  If so, can you:

- Tag this issue as `design`
- Comment on issue #37 saying that issue #37 is only around displaying **page 1** of `SAS COVID 19 infographics 1 2.pdf` and page two is covered by this one

**Comment by greggmiller** [comment]
Yes, that's true #37 is just Page 1 and the bullet points I entered there for step by step advice.
I created a duplicate issue of this and called it in infographics and detailed in #100 should we close one of them have provided more up to date details there?
I forgot that I had written the above last night so as I must have been quite tired. Apologies @lukesleeman @marcedwards

**Comment by galtaforce** [comment]
> Let me know @galtaforce if there are any other inforgraphics on airway I may have missed.
> 
> [Extubation_Infographic.pdf](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/files/4386891/Extubation_Infographic.pdf)
> [SAS COVID 19 infographics 1 2.pdf](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/files/4386892/SAS.COVID.19.infographics.1.2.pdf)
> 

That's all for now.

**Comment by lukesleeman** [comment]
Yep, looks like some confusion on whats in this screen.  Just to be 100% clear, it needs to display the following infographics:
**SAS COVID 19 infographics 1 2.pdf - Page 2**

![image](https://user-images.githubusercontent.com/902252/77730641-4cb9d680-7055-11ea-86c0-e04ddf8b077e.png)

**SAS COVID 19 infographics 1 2.pdf**

![image](https://user-images.githubusercontent.com/902252/77730676-5e02e300-7055-11ea-91db-3e15ef8557ae.png)

I think for the time being just displaying the infographics in some pinch to zoom thing would be fine - based on my discussions with @greggmiller it sounds like its important the orginal source images are there in an un-altered manner.

Tagging this as ready for design.  Let us know if you need more info

**Comment by greggmiller** [comment]
Yes I think it should also include: https://github.com/Western-Health-Covid19-Collaboration/source-documents/blob/master/COVID-19_Emergency_Intubation_Checklist.png

Although this seems like duplication from checklist #38 that @Mattro is working on,  some anaesthetists will probably want it infographics too. However, if cleaning up the image is significant work for @design lets just leave this out till someone asks for it or later iteration.

**Comment by lukesleeman** [comment]
Based on advice from the flutter devs, we are going to build a single re-usable infographic widget, then just configure it to display the different images used all over the app.

As such this issue has been rolled into issue #112 and I'm closing it.
